barretenberg/cpp/src/barretenberg/relations/translator_vm/translator_permutation_relation.hpp

```diff
@@ -14,14 +14,16 @@ template <typename FF_> class TranslatorPermutationRelationImpl {
         3  // left-shiftable polynomial sub-relation
     };
     /**
-     * @brief For ZK-Flavors: The degrees of subrelations considered as polynomials only in witness polynomials,
+     * @brief The degrees of subrelations considered as polynomials only in witness polynomials,
```

Comment:
Maybe it would be good to exemplify what you mean here to help the reader

Reply:
this is explained in the sumcheck docs, although I see your point  
do you think giving examples in relations' docs wouldn't pollute them? then I would also have to include those examples in every relation to make it uniform